stacks-common/src/address/c32.rs

```diff
@@ -69,44 +97,34 @@ fn c32_encode(input_bytes: &[u8]) -> String {
     String::from_utf8(result).unwrap()
 }
 
-fn c32_normalize(input_str: &str) -> String {
-    let norm_str: String = input_str
-        .to_uppercase()
-        .replace("O", "0")
-        .replace("L", "1")
-        .replace("I", "1");
```

Comment:
Reductions of the char scans and allocs here is one of the primary optimizations in this PR

Reply:
Where did the normalization code go?

Reply:
This normalization fn was no longer called once the encoding logic was changed to use a faster C32 alphabet lookup using the `C32_CHARACTERS_MAP` table. If you take a look at the code there, you'll see where the lowercase and special chars are included. I'm happy to add it back (will be dead code though).